apps/studio/src/components/sidebar/core/DatabaseDropdown.vue

```diff
@@ -5,12 +5,21 @@
       <a class="refresh" @click.prevent="refreshDatabases" :title="'Refresh Databases'">
         <i class="material-icons">refresh</i>
       </a>
+      <a class="refresh" @click.prevent="$modal.show('config-add-database')" :title="'Add Database'">
+        <i class="material-icons">add</i>
+      </a>
     </div>
+    <modal class="vue-dialog beekeeper-modal save-add-database" name="config-add-database" height="auto" :scrollable="true">
```

Comment:
We need to start using `<portal to="modals">` to wrap all modals so they render properly above other elements